lib/settings_base.py

```diff
@@ -80,6 +80,9 @@ def path(*folders):
 DATABASES['default']['TEST_CHARSET'] = 'utf8'
 DATABASES['default']['TEST_COLLATION'] = 'utf8_general_ci'
 
+# Run all views in a transaction unless they are decorated not to.
+ATOMIC_REQUESTS = True
```

Comment:
`ATOMIC_REQUESTS` is not a setting, it's a property of a database in `DATABASES`. We want that for the `default` (master) only I think.